features/docs/writing_support_code/after_hooks.feature

```diff
@@ -11,7 +11,7 @@ Feature: After Hooks
   Background:
     Given the standard step definitions
 
-  Scenario Outline: Retreive the status of a scenario as a symbol
+  Scenario Outline: Retrieve the status of a scenario as a symbol
```

Comment:
Please try and avoid doing this as part of an unrelated PR - better to put these kinds of changes into individual PRs, however small. :)